specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -516,42 +518,43 @@ scope kafka
         struct KafkaCreateTopic
         {
             string16 name;
+            int32 partitionCount;
             int16 replicas;
             KafkaPartition[] assignments;
             KafkaConfig[] configs;
         }
 
-        struct KafkaCreateTopicsRequestBeginEx
+        struct KafkaRequestCreateTopicsBeginEx
         {
             KafkaCreateTopic[] topics;
             int32 timeout;
             uint8 validateOnly;
         }
 
-        struct KafkaDeleteTopicsRequestBeginEx
+        struct KafkaRequestDeleteTopicsBeginEx
         {
             string16[] names;
             int32 timeout;
         }
 
         struct KafkaResource
         {
-            string16 name;
             KafkaResourceType type;
+            string16 name;
             KafkaConfig[] configs;
         }
 
-        struct KafkaAlterConfigsRequestBeginEx
+        struct KafkaRequestAlterConfigsBeginEx
         {
             KafkaResource[] resources;
             uint8 validateOnly;
         }
 
-        struct KafkaRequestBeginEx switch (uint8)
+        union KafkaRequestBeginEx switch (uint8)
         {
-            case 33 kafka::stream::KafkaAlterConfigsRequestBeginEx alterConfigs;
-            case 20: kafka::stream::KafkaDeleteTopicsRequestBeginEx deleteTopics;
-            case 19 kafka::stream::KafkaCreateTopicsRequestBeginEx createTopics;
+            case 33: kafka::stream::KafkaRequestAlterConfigsBeginEx alterConfigs;
+            case 20: kafka::stream::KafkaRequestDeleteTopicsBeginEx deleteTopics;
+            case 19: kafka::stream::KafkaRequestCreateTopicsBeginEx createTopics;
```

Comment:
```suggestion
            case 33: kafka::stream::KafkaAlterConfigsRequestBeginEx alterConfigs;
            case 20: kafka::stream::KafkaDeleteTopicsRequestBeginEx deleteTopics;
            case 19: kafka::stream::KafkaCreateTopicsRequestBeginEx createTopics;
```
I think this will read more naturally, and consistent with how we name for `KafkaBeginEx`, etc with `BeginEx` at the end.